tests/system/runtimes/test_nuclio.py

```diff
@@ -117,14 +120,89 @@ def test_serving_with_child_function(self):
 
         graph = function.set_topology("flow", engine="async")
 
+        # full_event defaults to True
         graph.to(
             ">>", "q1", path=f"v3io:///{self.stream_container}{self.stream_path}"
-        ).to(name="child", class_name="Identity", function="child")
+        ).to(name="child", class_name="Identity", function="child").to(
+            ">>",
+            "out",
+            path=f"/{self.stream_container}{self.stream_path_out}",
+            full_event=False,
+        )
+
+        graph.add_step(
+            name="otherchild", class_name="Augment", after="q1", function="otherchild"
+        )
 
-        function.add_child_function("child", child_code_path, "mlrun/mlrun")
+        graph["out"].after_step("otherchild")
+
+        function.add_child_function(
+            "child",
+            child_code_path,
+            image="mlrun/mlrun",
+        )
+        function.add_child_function(
+            "otherchild",
+            child_code_path,
+            image="mlrun/mlrun",
+        )
 
         self._logger.debug("Deploying nuclio function")
-        function.deploy()
+        url = function.deploy()
+
+        self._logger.debug("Triggering nuclio function")
+        resp = requests.post(url, json={"hello": "world"})
+        assert resp.status_code == 200
+
+        time.sleep(10)
```

Comment:
kind suggestion to replace it to `retry_until_successful` which will give you better coverage, as if there some network hiccups(you can give it 30sec as high threshold) the test won't fail, but it won't wait the whole 30sec if it succeeded earlier. Either way 10 sec is not such a big deal so I leave it to you to decide :)